test/spmd/tensor/test_dtensor.py

```diff
@@ -8,7 +8,8 @@
     DistTensorTestBase,
     with_comms,
 )
-from spmd.tensor import DeviceMesh, DTensor, Replicate, Shard, _Partial
+from spmd.tensor import DeviceMesh, DTensor
+from spmd.tensor.placement_types import _Partial, Replicate, Shard
```

Comment:
Move `Replicate` and `Shard` to `spmd.tensor` import maybe?

Reply:
it's test so I would like to keep placement type import be consistent because it also need to import `_Partial`.